<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemInfoWidget</class>
 <widget class="QWidget" name="ItemInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#m_TowerWidget {
    border: 3px solid grey;
}

QWidget#m_EnemyWidget {
    border: 3px solid grey;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="m_TowerWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color : blue;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_CostTowerTitle">
        <property name="text">
         <string>Cost:</string>
        </property>
        <property name="buddy">
         <cstring>m_TowerWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_CostTowerValue">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_PowerTowerTitle">
        <property name="text">
         <string>Power:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="m_PowerTowerValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_RangeTowerTitle">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="m_RangeTowerValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_UpgradeCostTowerTitle">
        <property name="text">
         <string>Upgrade:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="m_UpgradeCostTowerValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="m_SellCostTowerTitle">
        <property name="text">
         <string>Sell Cost:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="m_SellCostTowerValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_EnemyWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color : green;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="m_HealthEnemyTitle">
        <property name="text">
         <string>Health</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_BonusEnemyTitle">
        <property name="text">
         <string>Bonus:</string>
        </property>
        <property name="buddy">
         <cstring>m_EnemyWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_SpeedEnemyTitle">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="m_HealthEnemyValue">
        <property name="text">
         <string>0/15</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_BonusEnemyValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="m_SpeedEnemyValue">
        <property name="text">
         <string>Slow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
